src/Tools/Source/FakeSign/FakeSign.csproj

```diff
@@ -1,38 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yeah, I didn't think of staging the changes so there's no build break. I'll merge this, then remove the ref from the internal build, then delete the remnants.